polygeist/tools/cgeist/Test/Verification/imag_complex.c

```diff
@@ -0,0 +1,12 @@
+// RUN: cgeist %s --function=* -S | FileCheck %s
+// XFAIL: *
+
+#include <complex.h>
+
+int f2(int complex a) {
+  return __imag__(a);
+}
+
+float f3(float complex a) {
+  return __imag__(a);
+}
```

Comment:
complex not working at all, so this is out of scope.

Reply:
I'd would have wanted this test case to be merged with imag.c  but I see you need to XFAIL it for now (so it has to be a separate test).